tao_compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc

```diff
@@ -137,6 +137,18 @@ bool isWeightPrePackingEnabled() {
   return enabled;
 }
 
+bool isWeightPrePackingForMatMulEnabled() {
+  static bool enabled = []() {
+    const char* env = getenv("DISC_CPU_ENABLE_WEIGHT_PRE_PACKING_FOR_MATMUL");
+    if (!env) return true;
+    std::string envStr = env;
+    std::transform(envStr.begin(), envStr.end(), envStr.begin(),
+                   [](unsigned char c) { return std::tolower(c); });
+    return envStr == "true" || envStr == "1";
+  }();
+  return isWeightPrePackingEnabled() && enabled;
```

Comment:
Seems that this is for some senario where pre_packing for mamtul should be disabled while pre_packing for others (mostly conv) should be enabled?

Reply:
Yes. We rely on onednn to call ACL gemm while we call acl conv directly currently. Thus, we can only share packed weight across different shapes for conv op at the moment. We'll fix this later by bypassing onednn.

Reply:
I see. Thanks!